File: VB.Net/Input-Processing-and-Output/Mad-Lib/MadLib.vb
Public Class MadLib
    Private _noun As String = "{noun}"
    Private _verb As String = "{verb}"
    Private _adjective As String = "{adjective}"
    Private _adverb As String = "{adverb}"

    Private Sub TextBox1_TextChanged(sender As Object, e As EventArgs) Handles TextBox1.TextChanged, TextBox2.TextChanged, TextBox3.TextChanged, TextBox4.TextChanged
        _noun = TextBox1.Text
        _verb = TextBox2.Text
        _adjective = TextBox3.Text
        _adverb = TextBox4.Text

        RichTextBox1.Text = $"Did you know that a {_adjective} {_noun} can {_adverb} {_verb}? I know! It's crazy!"
    End Sub
End Class

File: C#/Input-Processing-and-Output/MadLib.cs
namespace CSharp.IPO
{
    internal class MadLib
    {
        internal string? noun, verb, adjective, adverb;
        internal MadLib() 
        {
            Console.Write("Enter a noun: ");
            noun = Console.ReadLine();

            Console.Write("Enter a verb: ");
            verb = Console.ReadLine();

            Console.Write("Enter an adjective: ");
            adjective = Console.ReadLine();

            Console.Write("Enter an adverb: ");
            adverb = Console.ReadLine();
        }

        // Create a simple mad-lib program that prompts for a noun,
        // a verb, an adverb, and an adjective
        // and injects those into a story that you create.
        public static void Main3()
        {
            MadLib madlib = new();

            /*
             • Use a single output statement for this program.
             • If your language supports string interpolation or string
               substitution, use it to build up the output.
            */
            Console.WriteLine($"Did you know that a {madlib.adjective} {madlib.noun} can {madlib.adverb} {madlib.verb}? I know! It's crazy!");
        }
    }
}
